src/OSSupport/Socket.cpp

```diff
@@ -1,377 +0,0 @@
-
```

Comment:
Oh, I misinterpreted you then. I was expecting a file to be added to the cmake file list, not the missing file to be removed ;)